azurecompute/src/main/java/org/jclouds/azurecompute/binders/DeploymentParamsToXML.java

```diff
@@ -50,21 +50,11 @@
                     .e("AdminPassword").t(params.password()).up()
                     .e("ResetPasswordOnFirstLogon").t("false").up()
                     .e("EnableAutomaticUpdate").t("false").up()
-                    .e("DomainJoin")
-                    .e("Credentials")
-                    .e("Domain").t(params.name()).up()
-                    .e("Username").t(params.username()).up()
-                    .e("Password").t(params.password()).up()
-                    .up() // Credentials
-                    .e("JoinDomain").t(params.name()).up()
-                    .up() // Domain Join
-                    .e("StoredCertificateSettings").up()
                     .e("WinRM")
                     .e("Listeners")
                     .e("Listener")
-                    .e("Protocol").t("http").up().up().up().up()
+                    .e("Protocol").t("Https").up().up().up().up()
```

Comment:
It is tricky a bit because the parameter is required only for windows, it is not something needed for ssh.
I tried adding `Boolean useWinrmOverHttps` to the params in `AutoValue_DeploymentParams` but then not sure it has to be presented to the parent `DeploymentParams`.
There are a lot of other places that the parameter has to be added.
AutoValue_DeploymentParams, DeploymentParams, AzureComputeServiceAdapter, AzureComputeTemplateOptions.